### PR TITLE
docs: fix linter errors for update-to-latest-version.md

### PR DESCRIPTION
**Pull request by dsnoeijer**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #47179

## What is the new behavior?
No more linter errors in the file.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/update-to-latest-version.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ For information about the deprecation and removal practices of Angular, see [Ang
 
 *   **Strictly Typed Reactive Forms**
 
-    The Reactive Forms types `AbstractControl`, `FormControl`, `FormGroup`, and `FormArray` now support a generic parameter which allows for strict typing of the controls. An automatic migration will convert existing usages of these types to special `Untyped` aliases which preserve the existing behavior.
+    The Reactive Forms types `AbstractControl`, `FormControl`, `FormGroup`, and `FormArray` now support a generic parameter which allows for strict typing of the controls. An automatic migration is going to convert existing usages of these types to special `Untyped` aliases which preserve the existing behavior.
```
Comment (bob-watson) [inline aio/content/guide/update-to-latest-version.md:19]: Better to convert it to present tense.
```suggestion
    The Reactive Forms types `AbstractControl`, `FormControl`, `FormGroup`, and `FormArray` now support a generic parameter which allows for strict typing of the controls. An automatic migration converts existing usages of these types to special `Untyped` aliases which preserve the existing behavior.
```

File: aio/content/guide/update-to-latest-version.md [hunk 2]
```diff
@@ -31,20 +31,20 @@ For information about the deprecation and removal practices of Angular, see [Ang
 | | Details |
 |:--- |:--- |
 | [**PR&nbsp;#45729**](https://github.com/angular/angular/pull/45729) | `initialNavigation: 'enabled'` was deprecated in v11 and is replaced by `initialNavigation: 'enabledBlocking'.`. |
-| [**PR&nbsp;#42803**](https://github.com/angular/angular/pull/42803) | Forms `email` input coercion: forms `email` input value will be considered as true if it is defined with any value rather than false and 'false'. |
-| [**PR&nbsp;#33729**](https://github.com/angular/angular/pull/33729) | Objects with a length key set to zero will no longer validate as empty. This is technically a breaking change, since objects with a key `length` and value `0` will no longer validate as empty. This is a very minor change, and any reliance on this behavior is probably a bug anyway. |
+| [**PR&nbsp;#42803**](https://github.com/angular/angular/pull/42803) | Forms `email` input coercion: forms `email` input value is going to be considered as true if it is defined with any value rather than false and 'false'. |
```
Comment (bob-watson) [inline aio/content/guide/update-to-latest-version.md:34]: In present tense:
```suggestion
| [**PR&nbsp;#42803**](https://github.com/angular/angular/pull/42803) | Forms `email` input coercion: forms `email` input value is considered `true` if it is defined with any value other than `false` or `'false'`. |
```

**Review by bob-watson — CHANGES_REQUESTED** [review]
Thanks for the edits!
The use of present tense in place of future tense is good.
My comments are mostly on the cases where one future-tense phase was exchanged for another.

**Comment by JeanMeche** [comment]
@dsnoeijer `update-to-latest-version.md` has been deleted, I think you can close your PR 👍

**Pull request closed without merging**